OrbitLinuxTracing/TracerThread.cpp

```diff
@@ -368,6 +368,20 @@ bool TracerThread::OpenTracepoints(const std::vector<int32_t>& cpus) {
       !OpenRingBuffersForTracepoint("task", "task_rename", cpus, &tracing_fds_, &task_rename_ids_,
                                     &tracepoint_ring_buffer_fds_per_cpu, &ring_buffers_);
 
+  TracepointInfoToHashSet instrumented_tracepoints_ids_;
+
+  for (const auto& selected_tracepoint : instrumented_tracepoints_) {
+    absl::flat_hash_set<uint64_t> empty_hash_set;
+    instrumented_tracepoints_ids_.emplace(selected_tracepoint, empty_hash_set);
+    tracepoint_event_open_errors |= !OpenRingBuffersForTracepoint(
+        selected_tracepoint.category().c_str(), selected_tracepoint.name().c_str(), cpus,
+        &tracing_fds_, &instrumented_tracepoints_ids_.at(selected_tracepoint),
+        &tracepoint_ring_buffer_fds_per_cpu, &ring_buffers_);
+
+    for (const auto& stream_id : instrumented_tracepoints_ids_.at(selected_tracepoint))
+      stream_id_to_tracepoint_.emplace(stream_id, selected_tracepoint);
```

Comment:
We use the braces {} also for one-line blocks.

Reply:
I understand. Thank you!